lms/envs/common.py

```diff
@@ -397,6 +397,9 @@
 
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
+
+    # Enable Next Button to jump sequences in Sequence Navigation bar.
+    'ENABLE_NEXT_BUTTON_ACROSS_SECTIONS': False,
```

Comment:
Note to self: This flag has [since been removed from master](https://github.com/edx/edx-platform/blame/master/lms/envs/common.py#L373), so should probably be removed from here too.